client/preprocessors.py

```diff
@@ -317,17 +317,17 @@ def run(self):
             for section in toc['sections']:
                 markdown += self.compile_section(project, section, 2)
             markdown = self.fix_links(markdown)
-            output_file = os.path.join(self.output_dir, '{0}.md'.format(project.identifier))
+            output_file = os.path.join(self.output_dir, '{0}-{1}.md'.format(idx+1, project.identifier))
             write_file(output_file, markdown)
 
             # Copy the toc and config.yaml file to the output dir so they can be used to
             # generate the ToC on live.door43.org
             toc_file = os.path.join(self.source_dir, project.path, 'toc.yaml')
             if os.path.isfile(toc_file):
-                copy(toc_file, os.path.join(self.output_dir, '{0}-toc.yaml'.format(project.identifier)))
+                copy(toc_file, os.path.join(self.output_dir, '{0}-{1}-toc.yaml'.format(idx+1, project.identifier)))
             config_file = os.path.join(self.source_dir, project.path, 'config.yaml')
             if os.path.isfile(config_file):
-                copy(config_file, os.path.join(self.output_dir, '{0}-config.yaml'.format(project.identifier)))
+                copy(config_file, os.path.join(self.output_dir, '{0}-{1}-config.yaml'.format(idx+1, project.identifier)))
```

Comment:
Since `idx` is never used without adding 1 to it, it might be more efficient to just add 1 at the beginning of the loop rather than doing it repeatedly.